### ISSUE TITLE
project - dependencies_pile_map broken

### ISSUE THREAD

**Issue by jens-erdmann** [vinland-technology/flict#164]
https://github.com/vinland-technology/flict/blob/4dab580a3436a955c16cc7b87459b18729596113/flict/flictlib/project.py#L138

The dependencies_pile_map function of project.py is either redundant or broken.

```
        dep_pile = self.dependencies_pile()
        dep_pile_json = []
        for d in dep_pile:
            dep_json = d
            dep_json['expanded_license'] = d['expanded_license']
            dep_pile_json.append(dep_json)
```

`dep_pile_json` is just a copy of `dep_pile`.

Either we implement a logic that flattens the list or we should replace `dependencies_pile_map()` calls with `dependencies_pile()`.

**Comment by priv-kweihmann** [comment]
Yes, in a strict pythonic sense we should go with 
```python
    def dependencies_pile_map(self):
        """Returns the pile ("flattened tree")
        of a project's dependencies as a map
        """
        dep_pile = self.dependencies_pile()
        dep_pile_json = set()
        for d in dep_pile:
            dep_pile_json.update(d.get('expanded_license', []))

        return dep_pile_json
```

**Comment by jens-erdmann** [comment]
Fixed by #209
